tpl/admin.tpl

```diff
@@ -0,0 +1,85 @@
+{# findologicxmlexport/tpl/admin.tpl for findologicxmlexport plugin #}
+
+<div class="plugin-findologicxmlexport">
+    <h1>{{languageText['menu']}}</h1>
+    <p>
+        {{languageText['youCan']}}<a href="lib/plugins/findologicxmlexport">{{languageText['callExport']}}</a>.
+    </p>
+    <fieldset style="margin-top: 5em;">
+        {% if pagesWithoutTitle %}
+        <div class="notify">
+            {{languageText['noTitleWarning']}} <br/> {{languageText['noTitleWarningMoreInformation']}}
+        </div>
+        {% else %}
+        <div class="success">
+            {{languageText['allPagesHaveATitle']}}
+        </div>
+        {% endif %}
+        <legend>{{languageText['pagesWithoutTitle']}} ({{pagesWithoutTitle|length}})</legend>
+        <div class="table">
+            {% if pagesWithoutTitle %}
+            <table class="table table-condensed">
+                <tbody>
+                <tr>
+                    <th>
+                        {{languageText['namespace']}}
+                    </th>
+                    <th>
+                        {{languageText['url']}}
+                    </th>
+                    <th>
+                        {{languageText['lasteditby']}}
+                    </th>
+                    <th>
+                        {{languageText['lastedited']}}
+                    </th>
+                    <th>
+                        {{languageText['edit']}}
+                    </th>
+                </tr>
+                {% set amount = 0 %}
+                {% set morePages = 0 %}
+                {% for pageWithoutTitle in pagesWithoutTitle %}
+                {% if amount < 5 %}
```

Comment:
The controller should determine how many pages are displayed before they are cut off. Keep the template as stupid as possible, it should just concern itself with building HTML.